### PR TITLE
Appbar now has gradient colors

### LINKED ISSUE #201: The app bar should have correct colors so that it fits the design guide

### PR DESCRIPTION
**Pull request by msimon16**
This closes #201.

The appbar now has gradient colors, according to the design guide.

### CODE REVIEW

File: lib/widgets/giraf_title_header.dart [hunk 1]
```diff
@@ -19,20 +19,24 @@ class GirafTitleHeader extends StatelessWidget implements PreferredSizeWidget {
           child: Container(
             child: Padding(
               padding: const EdgeInsets.fromLTRB(10, 10, 10, 10),
-              child: Center(child: Text(title, textAlign: TextAlign.center,)),
+              child: Center(
+                  child: Text(
+                title ?? '',
```
Comment (boginw) [inline lib/widgets/giraf_title_header.dart:24]: Test this line?

**Comment by KVISDAOWNER** [comment]
@aau-giraf/sw602f19  you have been assigned this. Here's your checklist. I marked @KVISDAOWNER  as the reviewer, but you might change it as you like.
Please use the following checklist. Feel free to check it off as you go along.
**Code Design**
- [ ] Is the code in the right place? (Both in terms of folder structure and class structure)
- [ ] Could this code, to the best of your knowledge, have reused existing code?
- [ ] Is the code over-engineered? Examples of over-engineering: Implemented behavior for future problems
- [ ] Does it introduce functionality that is not necessary for solving the problem?

**Code Readability and Maintainability**
- [ ] Are names meaningful and self-documenting?
- [ ] Is it understandable, by reading the code, what it does?
- [ ] Is the code simple and readable? i.e. it contains no “hack”, or obscure solution.
- [ ] Is it understandable, by reading the tests, what they do?
- [ ] Do the tests cover sensible cases? Both happy paths and exception paths?
- [ ] Do the tests cover the full functionality? i.e., will the tests fail if some of the requested functionality is missing?
- [ ] Are error messages in the code understandable?
- [ ] Are confusing sections of code covered by comments?

**Functionality**
For this part, open the app, and test:
- [ ] Are the features that the code claims to implement, actually implemented?
- [ ] Does it look like the code has bugs?
- [ ] Are all new screens reachable through in-app navigation?

**Points for Discussion and Reflection**
- [ ] Are UI messages, that the user might meet, actually helpful?
- [ ] Are there errors that are bad practice? Like, hard-coding a test database connection?
- [ ] Is there well written code in this? Remember to tell the developer!

**Review by boginw — CHANGES_REQUESTED** [review]
Almost perfect :)

**Comment by KVISDAOWNER** [comment]
@aau-giraf/sw611f19  you have been assigned this. Here's your checklist. I marked @boginw  as the reviewer, but you might change it as you like.
Please use the following checklist. Feel free to check it off as you go along.
**Code Design**
- [ ] Is the code in the right place? (Both in terms of folder structure and class structure)
- [ ] Could this code, to the best of your knowledge, have reused existing code?
- [ ] Is the code over-engineered? Examples of over-engineering: Implemented behavior for future problems
- [ ] Does it introduce functionality that is not necessary for solving the problem?

**Code Readability and Maintainability**
- [ ] Are names meaningful and self-documenting?
- [ ] Is it understandable, by reading the code, what it does?
- [ ] Is the code simple and readable? i.e. it contains no “hack”, or obscure solution.
- [ ] Is it understandable, by reading the tests, what they do?
- [ ] Do the tests cover sensible cases? Both happy paths and exception paths?
- [ ] Do the tests cover the full functionality? i.e., will the tests fail if some of the requested functionality is missing?
- [ ] Are error messages in the code understandable?
- [ ] Are confusing sections of code covered by comments?

**Functionality**
For this part, open the app, and test:
- [ ] Are the features that the code claims to implement, actually implemented?
- [ ] Does it look like the code has bugs?
- [ ] Are all new screens reachable through in-app navigation?

**Points for Discussion and Reflection**
- [ ] Are UI messages, that the user might meet, actually helpful?
- [ ] Are there errors that are bad practice? Like, hard-coding a test database connection?
- [ ] Is there well written code in this? Remember to tell the developer!

**Review by KVISDAOWNER — DISMISSED** [review]

File: lib/widgets/giraf_title_header.dart (hunk 1 above)
Comment (msimon16) [reply lib/widgets/giraf_title_header.dart:24]: Two tests have been made, testing that the given title is rendered, and a title that is null is replaced with ''

**Review by andpaacoke — APPROVED** [review]
Looks good. PO approved 👍

**Review by boginw — APPROVED** [review]

**Review by KVISDAOWNER — APPROVED** [review]

**Pull request merged**